rwatch/ui/animation/animation.h

```diff
@@ -35,7 +35,7 @@ typedef enum {
 } AnimationCurve;
 
 typedef void (*AnimationSetupImplementation)(struct Animation *animation);
-typedef void (*AnimationUpdateImplementation)(struct Animation *animation, const uint32_t time_normalized);
+typedef void (*AnimationUpdateImplementation)(struct Animation *animation, const int32_t time_normalized);
```

Comment:
we should use `AnimationProgress` here, just to keep style with the Pebble API

Reply:
agreed. likely NOT int32_t (unless you got time travel cracked)

Reply:
oh, I thought of that but then derped and got the types backward in my head.